scripts/debian/deb_test.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+set -x
+set -v
+
+echo "deb_setup starting within docker container"
+
+export GOPATH=${HOME}/go
+export PATH=${GOPATH}/bin:/usr/local/go/bin:${PATH}
+
+apt-get update
+apt-get install -y gnupg2 curl software-properties-common python3
+apt-key add /stuff/key.pub
+add-apt-repository -y "deb [trusted=yes] http://${DC_IP}:8111/ stable main"
+apt-get update --allow-unauthenticated
```

Comment:
shouldn't need `--allow-unauthenticated` now that we're building with a gpg key

Reply:
ok, removed